### PR TITLE
Add new Testnets

### PR DESCRIPTION
**Pull request by cryptomanz1**

### CODE REVIEW

File: blockchains/elastos/info/info.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Elastos",
+    "website": "https://elastos.info",
+    "description": "Elastos is an ecosystem of decentralized services supported by its own protocol blockchain with 50%+ of Bitcoin’s hashrate and multiple sidechains including a W3C Decentralized Identifier (DID) compliant identity chain and an industry standard Ethereum sidechain.",
```
Comment (apepayer) [inline blockchains/elastos/info/info.json:4]: elastos folder should be testnet related, where would you put mainnet then? also without mainnet why you decided to add this testnet?

File: blockchains/kovan/info/info.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Optimism Kovan",
+    "website": "https://kovan-testnet.github.io/website/",
+    "description": "Kovan is a Proof of Authority (PoA) publicly accessible blockchain for Ethereum; created and maintained by a consortium of Ethereum developers, to aide the Ethereum developer community.",
```
Comment (apepayer) [inline blockchains/kovan/info/info.json:4]: folder should be something like optimismkovan. as kovan can be on other chains

File: blockchains/rinkeby/info/info.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Rinkeby Testnet",
+    "website": "https://www.rinkeby.io/#stats",
+    "description": "The Rinkeby testnet is an Ethereum testnet that developers use to test decentralized applications before deploying them to the Ethereum mainnet.",
+    "explorer": "https://www.rinkeby.io/#explorer",
+    "symbol": "ETH",
+    "rpc_url": "https://rinkeby.boba.network",
```
Comment (apepayer) [inline blockchains/rinkeby/info/info.json:7]: why boba?

File: blockchains/kovan/info/info.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    "explorer": "https://kovan.ethplorer.io",
```
Comment (vcoolish) [inline blockchains/kovan/info/info.json:5]: folder should be something like optimismkovan. as kovan can be on other chains

File: blockchains/elastos/info/info.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    "explorer": "https://esc-testnet.elastos.io",
+    "research": "https://elastos.info",
```
Comment (vcoolish) [inline blockchains/elastos/info/info.json:6]: elastos folder should be testnet related, where would you put mainnet then? also without mainnet why you decided to add this testnet?

File: blockchains/rinkeby/info/info.json [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+    "type": "coin",
```
Comment (vcoolish) [inline blockchains/rinkeby/info/info.json:8]: why boba?

Comment (vcoolish) [reply blockchains/rinkeby/info/info.json:8]: it should me for ethereum

File: blockchains/ropstan/info/info.json [hunk 7]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Ropstan",
+    "website": " https://github.com/ethereum/ropsten",
+    "description": "The Ropsten testnet, which is now deprecated, is an Ethereum-based test blockchain used by Ethereum developers to test protocol upgrades before deploying applications on the Ethereum mainnet.",
+    "explorer": "https://ropsten.etherscan.io",
+    "research": "https://elastos.info",
+    "symbol": "ETH",
+    "rpc_url": "https://ropsten.infura.io/v3/%7BINFURA_API_KEY%7D",
```
Comment (vcoolish) [inline blockchains/ropstan/info/info.json:8]: is there any without API key?

File: blockchains/ropstan/info/info.json [hunk 8]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Ropstan",
+    "website": " https://github.com/ethereum/ropsten",
+    "description": "The Ropsten testnet, which is now deprecated, is an Ethereum-based test blockchain used by Ethereum developers to test protocol upgrades before deploying applications on the Ethereum mainnet.",
+    "explorer": "https://ropsten.etherscan.io",
+    "research": "https://elastos.info",
```
Comment (vcoolish) [inline blockchains/ropstan/info/info.json:6]: why elastos?

File: blockchains/ropstan/info/info.json [hunk 9]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Ropstan",
```
Comment (vcoolish) [inline blockchains/ropstan/info/info.json:2]:
```suggestion
    "name": "Ropsten",
```

File: blockchains/goerli/info/info.json [hunk 10]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Goerli",
+    "website": "https://goerlifaucet.com",
+    "description": "The Alchemy Goerli faucet is free, fast, and does not require authentication, though you can optionally login to Alchemy to get an increased drip.",
+    "explorer": "https://goerli.explorer.zksync.io",
```
Comment (vcoolish) [inline blockchains/goerli/info/info.json:5]: why is it related to zksync ?

**Pull request closed without merging**